zeth-contracts/contracts/Groth16Mixer.sol

```diff
@@ -86,6 +86,7 @@ contract Groth16Mixer is BaseMixer {
         // 2.a Verify the signature on the hash of data_to_be_signed
         bytes32 hash_to_be_signed = sha256(
             abi.encodePacked(
+                uint256(msg.sender),
```

Comment:
Why do you encode it as a uint256 and not a bytes32? Is there a reason in particular?

Reply:
I originally tried that, but solidity doesn't allow it for some reason.

Reply:
Yeah I can understand. For logs you have to use bytes32(uint256(uint())).